ansible/roles/couchdb/templates/couchdb.config.j2

```diff
@@ -5,4 +5,4 @@ COUCHDB_STDOUT_FILE=/dev/null
 COUCHDB_STDERR_FILE=/usr/local/var/log/couchdb/couch.err
 COUCHDB_RESPAWN_TIMEOUT=5
 COUCHDB_OPTIONS=
-ENV_ERL_FLAGS="+A 48"
+ENV_ERL_FLAGS="+A 16"
```

Comment:
for some reason when i try to examine how many threads there are, htop shows a number of threads that is always 20 more than what we put here. so when i saw it was 24 before i think we were legitimately only running 4. i bumped this to 16 for now so it's 1 per core